src/base/bittorrent/tracker.h

```diff
@@ -48,54 +52,58 @@ namespace BitTorrent
 {
     struct Peer
     {
-        QHostAddress ip;
         QByteArray peerId;
-        int port;
+        ushort claimedPort = 0;  // self-claimed by peer
```

Comment:
I would call it just "port" since there is no other port fields in this structure. Your comment is enough to describe it.

Reply:
done.